### PR TITLE
Add Vagrant file for creating Master and Slave VMs

### PR DESCRIPTION
**Pull request by TrimBiggs**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -22,7 +22,7 @@ ENV HOME /root
 # Caution: If the following two environment variables are set to a branch (i.e. master),
 # docker builds will need to be done with --no-cache. Failing to do so will cause  docker 
 # to not notice the latest commits pushed to said branch, and instead use a stale version of the branch.
-ENV MESOS_BRANCH 0.26.0
+ENV MESOS_BRANCH master
```
Comment (TrimBiggs) [inline Dockerfile]: Do we want to be on `master` here still?

**Comment by djosborne** [comment]
ok to test

File: Dockerfile (hunk 1 above)
Comment (djosborne) [inline Dockerfile]: No, we'll keep it at 0.26.0

File: Vagrantfile [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+  (1..num_instances).each do |i|
+    vm_name = "%s-%02d" % [instance_name_prefix, i]
+    config.vm.define vm_name do |host|
+      domain = "mesos.test"
+      host.vm.hostname = "%s.%s" % [vm_name, domain]
+
+      # Assign IP and prepend IP/hostname pair to /etc/hosts for correct FQDN IP resolution
+      ip = "172.18.8.#{i+100}"
+      host.vm.network :private_network, ip: ip
+      host.vm.provision :shell, :inline "echo '#{ip}  #{host.vm.hostname}' | cat - /etc/hosts > tmp && mv tmp /etc/hosts", :privileged => true
```
Comment (djosborne) [inline Vagrantfile]: Move the colon to the end of inline - should be `:shell, inline: "echo...`

**Comment by djosborne** [comment]
Closing in favor of #44

**Pull request closed without merging**